Sadly doesn’t seem to fix the build:

```
/nix/store/isgsi0yl9z8g2r9iaixh1ziszf47apm4-bash-5.2p37/bin/bash ../libtool --silent --tag=CC   --mode=compile clang -DHAVE_CONFIG_H -I. -I..    -I/nix/store/q7bngy6aghr8z9hsx0mj2bnkmri1nknw-xorgproto-2024.1/include -I/nix/store/0f342khrmnp62q99ak3s0ghrym5d3w00-libX11-1.8.10-dev/include -I/nix/store/p2fycb7yz6zh0pqc0bkrss2s7b06wfas-libXext-1.3.6-dev/include  -Wall -Wpointer-arith -Wstrict-prototypes -Wmissing-prototypes -Wmissing-declarations -Wnested-externs -fno-strict-aliasing -Wbad-function-cast -Wformat=2  -F/System/Library/Frameworks/ApplicationServices.framework/Frameworks -I../include -I../include/X11 -I../include/X11/extensions -g -O2 -c -o applewm.lo applewm.c
applewm.c:43:10: fatal error: 'HIServices/Processes.h' file not found
#include <HIServices/Processes.h>
         ^~~~~~~~~~~~~~~~~~~~~~~~
applewm.c:43:10: note: did not find header 'Processes.h' in framework 'HIServices' (loaded from '/System/Library/Frameworks/ApplicationServices.framework/Frameworks')
1 error generated.
```

Unclear to me whether it’s loading it from the SDK and that’s still not enough, or whether `-F` is actually not being interpreted relative to the `SDKROOT` for some reason. Will try to investigate.

--- a/pkgs/servers/x11/xorg/default.nix
+++ b/pkgs/servers/x11/xorg/default.nix
@@ -1018,11 +1018,11 @@ self: with self; {
   # THIS IS A GENERATED FILE.  DO NOT EDIT!
   libAppleWM = callPackage ({ stdenv, pkg-config, fetchurl, xorgproto, libX11, libXext, testers }: stdenv.mkDerivation (finalAttrs: {
     pname = "libAppleWM";
-    version = "1.4.1";
+    version = "be972ebc3a97292e7d2b2350eff55ae12df99a42";
     builder = ./builder.sh;
     src = fetchurl {
-      url = "mirror://xorg/individual/lib/libAppleWM-1.4.1.tar.bz2";
-      sha256 = "0r8x28n45q89x91mz8mv0zkkcxi8wazkac886fyvflhiv2y8ap2y";
+      url = "https://gitlab.freedesktop.org/xorg/lib/libAppleWM/-/archive/be972ebc3a97292e7d2b2350eff55ae12df99a42/libAppleWM-be972ebc3a97292e7d2b2350eff55ae12df99a42.tar.bz2";
+      sha256 = "1hrq03pahmrbb05r6a7j7m1nxl65wlfi6d2lwm1kvra63q91f9ph";
     };
     hardeningDisable = [ "bindnow" "relro" ];
     strictDeps = true;

--- a/pkgs/servers/x11/xorg/overrides.nix
+++ b/pkgs/servers/x11/xorg/overrides.nix
@@ -169,10 +169,8 @@ self: super:
   });
 
   libAppleWM = super.libAppleWM.overrideAttrs (attrs: {
-    buildInputs = attrs.buildInputs ++ [ ApplicationServices ];
-    preConfigure = ''
-      substituteInPlace src/Makefile.in --replace -F/System -F${ApplicationServices}
-    '';
+    nativeBuildInputs = attrs.nativeBuildInputs ++ [ autoreconfHook ];
+    buildInputs =  attrs.buildInputs ++ [ xorg.utilmacros ];
   });
 
   libXau = super.libXau.overrideAttrs (attrs: {

--- a/pkgs/servers/x11/xorg/tarballs.list
+++ b/pkgs/servers/x11/xorg/tarballs.list
@@ -165,7 +165,7 @@ mirror://xorg/individual/font/font-sun-misc-1.0.4.tar.xz
 mirror://xorg/individual/font/font-util-1.4.1.tar.xz
 mirror://xorg/individual/font/font-winitzki-cyrillic-1.0.4.tar.xz
 mirror://xorg/individual/font/font-xfree86-type1-1.0.5.tar.xz
-mirror://xorg/individual/lib/libAppleWM-1.4.1.tar.bz2
+https://gitlab.freedesktop.org/xorg/lib/libAppleWM/-/archive/be972ebc3a97292e7d2b2350eff55ae12df99a42/libAppleWM-be972ebc3a97292e7d2b2350eff55ae12df99a42.tar.bz2
 mirror://xorg/individual/lib/libdmx-1.1.5.tar.xz
 mirror://xorg/individual/lib/libfontenc-1.1.8.tar.xz
 mirror://xorg/individual/lib/libFS-1.0.10.tar.xz

